data.sql

```diff
@@ -1,8 +1,18 @@
 /* Populate database with sample data. */
 
 INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts)
-values
+VALUES
 ('Agumon', 'Feb 3, 2020', 10.23,'true', 0 ),
 ('Gabumon', 'Nov 15, 2018', 8, 'true', 2),
 ('Pikachu', 'Jan 7, 2021', 15.04, 'false', 1),
 ('Devimon', 'May 12, 2017', 11, 'true', 5);
+
+INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES
+('Charmander', 'Feb 8, 2020', 11,'false', 0 ),
+('Plantmon', 'Nov 15, 2021', 5.7,'true', 2 ),
+('Squirtle', 'Apr 2, 1993', 12.13, 'false', 3 ),
+('Angemon',  'Jun 12, 2005', 45,  'true', 1),
```

Comment:
- [ ] The weight of these 4 animals suppose to be a negative value, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/222268173-09c44bd9-5bee-4e71-a38e-afb585ae778f.png)

Reply:
@Omar-Muhamad Thank you so much for the observation, am on it.